### PR TITLE
[IN-2274] enable accessibility for xcode, need some explanation will ad…

### PR DESCRIPTION
**Pull request by lmesz-bitrise**
…d later

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [ ] The tool I added is stable, and __does not require frequent updates__. _Preinstalled tools on the LTS stacks
  are not updated, so if the tool requires frequent updates you should handle the installation on-demand (see: [Install Any Additional Tool - DevCenter](https://bitrise-io.github.io/devcenter/tips-and-tricks/install-additional-tools/) for more information)_
- [ ] I updated the corresponding tests if there were any.
- [ ] I added a version report line to [`system_report.sh`](/system_report.sh) for the new tool(s) I added
- [ ] I have added an entry to the CHANGELOG.md with a revisionID (the current date and the number of the change e.g.: 2019_10_11_1) and a brief description of the actual change
- [ ] I have updated the revisionID in [bitrise_profile](roles/profiles/files/bitrise_profile) **BITRISE_OSX_STACK_REV_ID** with the one I just created in the CHANGELOG.md

### CODE REVIEW

File: roles/install-xcode/tasks/main.yml [hunk 1]
```diff
@@ -27,3 +27,7 @@
   environment:
     XCODE_INSTALL_USER: "{{ XCODE_INSTALL_USER }}"
     XCODE_INSTALL_PASSWORD: "{{ XCODE_INSTALL_PASSWORD }}"
+
```
Comment (bitrise-tom) [inline roles/install-xcode/tasks/main.yml:30]: can we add a condition here for this to be for 10.15.6? I don't want this to do odd things when big sur comes along

Comment (lmesz-bitrise) [reply roles/install-xcode/tasks/main.yml:30]: I agree. Is it okay to fail if the os is not 10.15.6?

Comment (bitrise-tom) [reply roles/install-xcode/tasks/main.yml:30]: failing is a good idea, so we fix it

Comment (peterheja-bitrise) [reply roles/install-xcode/tasks/main.yml:30]: +1

**Review by peterheja-bitrise — CHANGES_REQUESTED** [review]
Do the big sur fail change

File: roles/install-xcode/tasks/main.yml (hunk 1 above)
Comment (lmesz-bitrise) [reply roles/install-xcode/tasks/main.yml:30]: Ahhh I totally forget that PR. Let me fix the PR and merge.

Comment (lmesz-bitrise) [reply roles/install-xcode/tasks/main.yml]: Failure added

**Review by fehersanyi-bitrise — APPROVED** [review]

File: roles/install-xcode/tasks/main.yml [hunk 2]
```diff
@@ -43,3 +43,12 @@
 #   shell: xcodebuild test -project bitrise-mac.xcodeproj -scheme bitrise-mac -destination 'platform=macOS,arch=x86_64,id=420E1727-1771-77C5-658D-E41EAF10AE6C'
 #   args:
 #     chdir: "/opt/bitrise/testapps/bitrise-mac"
+# TODO: We should use profiles for enable accessibility but it needs more time and investigation, will happen in the upcoming sprint
```
Comment (MrBlaise) [inline roles/install-xcode/tasks/main.yml:46]: Is this comment still true?

Comment (MrBlaise) [reply roles/install-xcode/tasks/main.yml:46]: Nevermind I see it in slack.

Comment (fehersanyi-bitrise) [reply roles/install-xcode/tasks/main.yml:46]: this is the truest of them all, we should do it via a plist file but we couldn't figure it out yet

File: roles/install-xcode/tasks/main.yml [hunk 3]
```diff
@@ -43,3 +43,12 @@
...
+- name: Enable accessibility to xcode
+  script: install_accessibility.sh
+  become: true
+
+- name: Fail if we are not on Catalina
```
Comment (MrBlaise) [inline roles/install-xcode/tasks/main.yml:51]: Shouldn't we fail before the enable accessibility task?

Comment (lmesz-bitrise) [reply roles/install-xcode/tasks/main.yml]: True, install take a while, moved upward.

**Review by fehersanyi-bitrise — DISMISSED** [review]

**Review by MrBlaise — APPROVED** [review]

**Review by MrBlaise — APPROVED** [review]

**Review by peterheja-bitrise — APPROVED** [review]

**Pull request merged**